lib/apiservers/service/swagger.json

```diff
@@ -100,6 +100,47 @@
         }
       }
     },
+    "/target/{target}/vch/{vchId}/certificate": {
```

Comment:
nit-pick: can you move this down just a hair, so that it follows `/target/{target}/vch/{vchId}`?